packages/console/src/consts/plan-quotas.ts

```diff
@@ -96,7 +106,7 @@ const enterprisePlanTable: SubscriptionPlanTable = {
   appLogoAndFaviconEnabled: true,
   darkModeEnabled: true,
   i18nEnabled: true,
-  mfaEnabled: true,
+  mfaEnabled: undefined,
```

Comment:
Should we use `isDevFeaturesEnabled` to keep the backward compatibility?

Reply:
Yes, updated.